storage/gc_queue_test.go

```diff
@@ -304,6 +305,13 @@ func TestGCQueueIntentResolution(t *testing.T) {
 	const now int64 = 48 * 60 * 60 * 1E9 // 2d past the epoch
 	tc.manualClock.Set(now)
 
+	// TODO: why is this needed to trigger intent resolution?
```

Comment:
`gcQueue` does two kinds of GC: old MVCC versions and transactions/intents. However, if there are no MVCC versions to GC it bails out early and doesn't attempt to clean up any intents: https://github.com/cockroachdb/cockroach/blob/bd8b2de68d85ae6af495826f17f8daae48d2f06d/storage/gc_queue.go#L218

I think we just need to rearrange that code in `gc_queue.go`: move the "Process push transactions in parallel" block up above "Set start and end keys".